t/ui/16-activity-view.t

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use OpenQA::Test::TimeLimit '20';
+use OpenQA::Test::Case;
+use OpenQA::Client;
+
+use OpenQA::SeleniumTest;
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl');
```

Comment:
do you think you could try without relying on these fixtures at all and just create which ones you need?

Reply:
I feel like that's a lot more code for almost the same result 🤔

Reply:
Unless you could point me to a very easy way of creating a job of each type

Reply:
fine, leave it.